### ISSUE TITLE
PHP Fatal error:  Uncaught Error: Undefined constant "Activitypub\Model\ACTIVITYPUB_SHARED_INBOX_FEATURE" in activitypub/includes/model/class-user.php:232

### ISSUE THREAD

**Issue by janboddez** [Automattic/wordpress-activitypub#635]
### Quick summary

Latest `master` gives me this error.

### Steps to reproduce

1. Install latest master branch.
2. Wait for a request to hit the inbox endpoint? I think? Or maybe it already happens when the REST API endpoints are being registered.

### What you expected to happen

That no such error occurs.

### What actually happened

Not sure what the exact effect is, yet, but the error regularly pops up in my `debug.log`.

### Impact

Some (< 50%)

### Available workarounds?

Yes, difficult to implement

### Logs or notes

I'm guessing this would be solved by referring to `\ACTIVITYPUB_SHARED_INBOX_FEATURE` instead, as it's defined as a global constant.

Using PHP 8.1, by the way.

**Comment by janboddez** [comment]
Not sure why this doesn't happen for other constants used similarly ...

**Comment by pfefferle** [comment]
Can you check your root `activitypub.php` file if it is defined there?

**Comment by janboddez** [comment]
It is!

**Comment by pfefferle** [comment]
🤔

**Comment by pfefferle** [comment]
And it's only the shared inbox const that throws an error?

**Comment by pfefferle** [comment]
Do you have a stack trace of the error?

Does the author page JSON load fine?

**Comment by janboddez** [comment]
I no longer have the stack trace. I also haven't seen the error in a little while, it may have been a temporary thing (I moved from v1.3.0 to master). (Op)cache-related, maybe? It's weird because I saw it a few more times even after deleting the complete plugin folder + reinstalling.

I also don't think anything else would load the user class without first including the main plugin file.

I'll keep an eye on it.

> I'm guessing this would be solved by referring to `\ACTIVITYPUB_SHARED_INBOX_FEATURE` instead, as it's defined as a global constant.

I though this might be the case because of the explicit namespace in the error message, but I think that's a PHP thing. That it'll look for a namespace constant/function but fall back to the global one. If it can't find neither, the error always mentions the namespaced version. Either way, I don't think this is a solution.

Either way, think I'll just close this one; I'll reopen it if it happens again.
